### PR TITLE
Write a sitemap generator for polygigs

### LINKED ISSUE #11: SEO - Sitemap

### PR DESCRIPTION
**Pull request by adamduren**
Closes #11.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -3,18 +3,18 @@
   "description": "A website for Polymer developers and employers to find each other.",
   "main": "index.html",
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.4.0",
```
Comment (adamduren) [inline bower.json:6]: Yarn must have done this.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (adamduren) [inline package.json:1]: Pretty standard and needed to install node packages.

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+const gulp = require('gulp');
```
Comment (adamduren) [inline gulpfile.js:1]: This turned out to be simple enough.

**Review by zacharytamas — APPROVED** [review]
Do we need to filter by `isActive`? I figure we'll clean it up regularly.

**Comment by adamduren** [comment]
Currently you can still get to the post by direct url. Maybe we'll want to change the view to say it's no longer available? I'll remove it so the searchengine maybe loses it.

**Comment by adamduren** [comment]
I'm also going to add a story for an RSS generator. That'll be convenient when we start getting more leads.

**Pull request merged**